src/sentry/issues/endpoints/group_events.py

```diff
@@ -40,7 +40,7 @@ class GroupEventsError(Exception):
 @region_silo_endpoint
 class GroupEventsEndpoint(GroupEndpoint, EnvironmentMixin):
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.PUBLIC,
```

Comment:
it's public and already documented here: https://docs.sentry.io/api/events/list-an-issues-events/